packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue

```diff
@@ -442,6 +459,43 @@ const onSelection = (id: string) => {
 	emit('select-output', id);
 };
 
+const isSaveDisabled = computed(() => {
+	const id = amr.value?.id;
+	if (!id || _.isEmpty(selectedOutputId.value)) return true;
+	const projectAssets = useProjects().activeProject.value?.projectAssets;
+	const outputPort = props.node.outputs?.find((port) => port.id === selectedOutputId.value);
+
+	return projectAssets?.some((asset) => asset.assetId === outputPort?.value?.[0]);
+});
+
+function updateNodeOutput(model: Model) {
+	if (!selectedOutputId.value || !model) return;
+
+	amr.value = model;
+	const outputPort = _.cloneDeep(props.node.outputs?.find((port) => port.id === selectedOutputId.value));
+
+	if (!outputPort) return;
+	outputPort.label = model.header.name;
+	outputPort.value = [model.id];
```

Comment:
I presume the model.id hasn't changed? If so I wouldn't bother updating this field.

Reply:
removed